workflow/deploy.md

```diff
@@ -0,0 +1,150 @@
+# Deploy to Kubernetes
+
+This section will present how to deploy a new service to our instance of Kubernetes.
+
+> To deploy a new website you need to be sure that you are **not** using `bower` and the the run script uses the version **1.*** of yarn.
```

Comment:
Could we use a different style for this block? I think quotation blocks should only be used for quoted text.

I think in this case it would probably be fine to just leave it as a normal paragraph, but if you really want it to stand out, you could italicise the whole block with `*my paragraph*`

Reply:
I made it stand out because we had this issue with @anthonydillon and I just wanted to insist on it. But ok I will remove the quote and italicise it.